includes/class-sensei-lesson.php

```diff
@@ -2214,24 +2215,75 @@ public function enqueue_styles( $hook ) {
 	} // End enqueue_styles()
 
 	/**
-	 * Add column headings to the "lesson" post list screen.
+	 * Add column headings to the "lesson" post list screen,
+	 * while moving the existing ones to the end.
 	 *
-	 * @access public
+	 * @access private
 	 * @since  1.0.0
-	 * @param  array $defaults
-	 * @return array $new_columns
+	 * @param  array $defaults  Array of column header labels keyed by column ID.
+	 * @return array            Updated array of column header labels keyed by column ID.
 	 */
 	public function add_column_headings( $defaults ) {
-		$new_columns                        = array();
+		$new_columns                        = [];
 		$new_columns['cb']                  = '<input type="checkbox" />';
 		$new_columns['title']               = _x( 'Lesson Title', 'column name', 'sensei-lms' );
 		$new_columns['lesson-course']       = _x( 'Course', 'column name', 'sensei-lms' );
 		$new_columns['lesson-prerequisite'] = _x( 'Pre-requisite Lesson', 'column name', 'sensei-lms' );
 		if ( isset( $defaults['date'] ) ) {
 			$new_columns['date'] = $defaults['date'];
 		}
+
+		// Make sure other sensei columns stay directly behind the new columns.
+		$other_sensei_culumns = [
```

Comment:
Same typo here`$other_sensei_culumns` -> `$other_sensei_columns`.

Reply:
Thank you, corrected.